p3/apps.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+
+
+class P3Config(AppConfig):
+    name = 'p3'
+
+    def ready(self):
+        import p3.listeners  # noqa
```

Comment:
This is the recommended way to import signals into django. Previously, this was imported at the end of models.py.